### DISCUSSION TITLE
Are the example datasets available anywhere else?

### DISCUSSION THREAD

**Discussion by carajillu** [mir-group/nequip#405] [General]
Dear all,

[EDIT] The minimal_stress.yaml example does work, as it downloads the dataset from https://qmml.org/ rather than quantum-machine.org

I have been trying to run the google colab tutorial, but all configs fail since they need to download the training datasets from quantum-machine.org, which seems to be down at the moment. Are those datasets available anywhere else? Or any others that I can use to test the different configurations? 

Thanks!

**Comment by Linux-cpp-lisp** [comment]
Hi @carajillu ,

Thanks for reporting this--- quantum-machine.org seems to be up and working right now for me, hopefully it was a temporary issue.
